src/lib/paginator/_paginator-theme.scss

```diff
@@ -0,0 +1,40 @@
+@import '../core/theming/palette';
+@import '../core/theming/theming';
+@import '../core/typography/typography-utils';
+
+
+@mixin mat-paginator-theme($theme) {
+  $foreground: map-get($theme, foreground);
+
+  .mat-paginator,
+  .mat-paginator-page-length-select .mat-select-trigger {
+    color: mat-color($foreground, secondary-text);
+  }
+
+  .mat-paginator-increment,
+  .mat-paginator-decrement {
+    border-top: 2px solid mat-color($foreground, 'icon');
+    border-right: 2px solid mat-color($foreground, 'icon');
+  }
+
+  .mat-icon-button[disabled] .mat-paginator-increment,
```

Comment:
You can nest this to avoid repetition:

```
.mat-icon-button[disabled] {
  .mat-paginator-increment,
  .mat-paginator-decrement {
    border-color: mat-color($foreground, 'disabled');
  }  
}
```